stock_location_last_inventory_date/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+{
+    "name": "Stock Location Last Inventory Date",
+    "summary": "Show the last inventory date for a leaf location",
+    "version": "13.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Warehouse Management",
```

Comment:
Please set to 'Warehouse' to align https://odoo-community.org/shop categories (Odoo standard ones)

Reply:
Done.